src/server/models/parking-lot.ts

```diff
@@ -0,0 +1,31 @@
+import { Document, Schema, model, Model } from 'mongoose';
+
+export interface ParkingLot extends Document {
+  name: string;
+}
+
+export const ParkingLotSchema = new Schema<
+  ParkingLot,
+  Model<ParkingLot, ParkingLot>
+>(
+  {
+    name: {
+      type: String,
+      required: true,
+    },
+  },
+  {
+    versionKey: false,
+  },
```

Comment:
Aren't we going to use `versionKey`? If thats the case, perhaps we should do the same for `UserModel` 🤔

Reply:
No, we should to set versionKey as false

Reply:
I should to set `versionKey` as true in UserModel? In this PR?

Reply:
Its fine! I can do it when I have a chance!